ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/README.md

```diff
@@ -63,6 +63,15 @@ BLOCK cmp-adaptiveform-container
     ELEMENT cmp-adaptiveform-container__wrapper
 ```
 
+In edit mode, the BEM structure includes a modifier for the cmp-adaptiveform-container block,
```

Comment:
In Edit mode, we utilize a distinct BEM modifier to indicate the editing state. To ensure simplicity for clients, it is crucial to maintain a consistent JSON payload. While properties can be added or removed based on the JSON view, the values of a property should remain consistent across various tiers, else client's (like EDS) would have to contribute to our code base by whitelisting things which is not scalable